test/e2e/rule_test.go

```diff
@@ -197,32 +176,39 @@ func (m *mockAlertmanager) ServeHTTP(resp http.ResponseWriter, req *http.Request
 // a mocked server instead of the "real" Alertmanager service.
 // The other end-to-end tests exercise against the "real" Alertmanager
 // implementation.
-func TestRuleAlertmanagerHTTPClient(t *testing.T) {
-	a := newLocalAddresser()
+func TestRule_AlertmanagerHTTPClient(t *testing.T) {
+	t.Skip("TODO: Allow HTTP ports from binaries running on host to be accessible.")
```

Comment:
Is this `skip` intentional or forgotten? If it's intentional better to create an issue, right?
Otherwise, we're skipping a test that previously functional.

Reply:
We are indeed.